docs/install-kubefed-operator.md

```diff
@@ -0,0 +1,24 @@
+# Installing the KubeFed Operator
+
+## Installing from the OperatorHub using the web console
+
+This procedure explains how to install and subscribe to the KubeFed Operator from the OperatorHub using the OpenShift Container Platform web console.
+
+### Prerequisites
+- Access to an OpenShift Container Platform cluster using an account with `cluster-admin` permissions.
```

Comment:
This doesn't have to be necessarily OCP. It can be OKD too right?

Reply:
OCP is sufficient for now. OKD is a thing but in a bit of an interstitial state; we can add it later.